README.md

```diff
@@ -157,10 +157,11 @@ Install this project with:
 
 ### Usage
```

Comment:
- [ ] [**OPTIONAL**] Good job adding documentation of your project, however i will suggest you to use some techniques to make your README file more professional :+1:  
User "``" to highlight something 

### Example

`npx eslint .`

User 
"```bash"
your code ...
"```"

### Example

```bash
npx exlint .
```
What do you think about these tips? :smile:

Reply:
thanks I will consider it